From 5473ae461408dbb183d7e8b2178d9338c43826d1 Mon Sep 17 00:00:00 2001
From: Tilmann Meyer <me@atiltedtree.dev>
Date: Thu, 18 Nov 2021 15:48:11 +0100
Subject: [PATCH 14/20] add header needed for IsGDKWaylandDisplay

---
 widget/gtk/WindowSurfaceProvider.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/widget/gtk/WindowSurfaceProvider.cpp b/widget/gtk/WindowSurfaceProvider.cpp
index 059aa9722e111..c23bd6e752558 100644
--- a/widget/gtk/WindowSurfaceProvider.cpp
+++ b/widget/gtk/WindowSurfaceProvider.cpp
@@ -10,6 +10,7 @@
 #include "mozilla/gfx/Logging.h"
 #include "mozilla/layers/LayersTypes.h"
 #include "nsWindow.h"
+#include "WidgetUtilsGtk.h"
 
 #ifdef MOZ_WAYLAND
 #  include "mozilla/StaticPrefs_widget.h"
-- 
2.35.1

