From 672b2d1f31a29a3efb1fc7cfabfd25bdbdc47b83 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Sun, 12 Dec 2021 02:16:02 +0100
Subject: [PATCH 21/23] bmo#1745560: Fix building against >=wayland-1.20

Signed-off-by: Thomas Deutschmann <whissi@gentoo.org>
---
 widget/gtk/mozwayland/mozwayland.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/widget/gtk/mozwayland/mozwayland.c b/widget/gtk/mozwayland/mozwayland.c
index 8dcf5c9c18fd6..3e71e44b66df6 100644
--- a/widget/gtk/mozwayland/mozwayland.c
+++ b/widget/gtk/mozwayland/mozwayland.c
@@ -39,6 +39,13 @@ MOZ_EXPORT void wl_event_queue_destroy(struct wl_event_queue* queue) {}
 
 MOZ_EXPORT void wl_proxy_marshal(struct wl_proxy* p, uint32_t opcode, ...) {}
 
+MOZ_EXPORT struct wl_proxy* wl_proxy_marshal_flags(
+    struct wl_proxy* proxy, uint32_t opcode,
+    const struct wl_interface* interface, uint32_t version, uint32_t flags,
+    ...) {
+  return NULL;
+}
+
 MOZ_EXPORT void wl_proxy_marshal_array(struct wl_proxy* p, uint32_t opcode,
                                        union wl_argument* args) {}
 
-- 
2.34.1

